app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -507,6 +510,13 @@ public function setDescription($description)
         return $this;
     }
 
+    public function setId(int $id): Email
```

Comment:
Why is this method needed?

Reply:
it was used in other email builder subscriber but that subscriber is not used in mautic as community uses grapesJS  for email builder if we will remove it from here then it will break that subscriber after unforking when we will use mautic repo for core code.

Reply:
I see. Looking at the usages on our fork, it's also used in some tests that we'll push later. So let's get this in.